osbs/conf.py

```diff
@@ -510,3 +511,18 @@ def get_low_priority_node_selector(self):
             nodeselector = dict([k.strip(), v.strip()] for (k, v) in raw_nodeselector.items())
 
         return nodeselector
+
+    def get_equal_labels(self):
+        equal_labels = []
+        equal_labels_str = self._get_value("equal_labels", self.conf_section,
+                                           "equal_labels")
+        if equal_labels_str:
+            if re.match('^[^:, ]+(:[^:, ]+)+\s*(,\s*[^:, ]+(:[^:, ]+\s*)+)*$', equal_labels_str):
```

Comment:
I feel like we should either use a regular expression to validate and parse the value, or else use string manipulation functions to validate and parse, rather than re to validate and funcs to parse.

If we *do* use a regular expression, I'd like a comment explaining what the intention behind it is. :-)